docs/includes/usage-examples/FindOneTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use MongoDB\Laravel\Tests\TestCase;
+
+use function print_r;
+
+class FindOneTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function updateOne(): void
+    {
+        require_once __DIR__ . '/Movie.php';
+
+      // begin-find-one
+        $movie = Movie::where('directors', 'Rob Reiner')
+          ->orderBy('_id')
+          ->first();
+
+        print_r($movie->toJson());
```

Comment:
Do we really want to display the results with `print_r`? That is not very compatible with PHPUnit.

Reply:
How would you prefer to display the results? Originally I used a view, but I changed it to `print_r` for simplicity

Reply:
Since it's a simple string, I modified to use `echo`.